packages/plugin-lighthouse/src/lib/runner/utils.ts

```diff
@@ -38,43 +39,46 @@ export class LighthouseAuditParsingError extends Error {
   }
 }
 
+function formatBaseAuditOutput(lhrAudit: Result): AuditOutput {
+  const { id: slug, score, numericValue, displayValue } = lhrAudit;
+  return {
+    slug,
+    score: score ?? 1, // score can be null
+    value: numericValue ?? score ?? 0, // not every audit has a numericValue
+    displayValue:
+      displayValue ??
+      (score === 1 ? 'passed' : score === 0 ? 'failed' : undefined),
```

Comment:
I just realized (from your test example) that Lighthouse uses `scoreDisplayMode: 'binary'` :open_mouth: 

That seems like a more reliable indicator for when it would be appropriate to display `passed` or `failed`.

```suggestion
      (scoreDisplayMode === 'binary'
        ? (score === 1
          ? 'passed'
          : 'failed')
        : undefined),
```